src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -30,6 +30,13 @@ public class GenerateConfig extends BaseCommand {
       description = "Directory to put generated cromwell.conf in. Defaults to current directory.")
   public String dir;
 
+  @CommandLine.Option(
```

Comment:
I *think* this is a good flag to add.

This will be interesting to see if we find we are slowly but surely adding more and more flags to this command because Cromwell is a big application with lots of possible settings.

The bucket name flag makes a lot of sense because it is a requirement. This one isn't required, but I agree that it is nice to have some built-in guardrails for our users.

Reply:
I think the good news is our CLI doesn't block users from configuring Cromwell however they want (users can fairly easily just change the config file directly). I think the main reason to add flags is for automated scripts and Getting Started notebooks that need programmatic access to the config file. 

I would like to add one more flag for configuring cromwell to communicate with a local database. And I can see users possibly needing options for things like call-caching, log directories, possibly others. But hopefully Terra has a better answer for workflows before "too many cli options" is actually a problem.